.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
-        id: generate-release-notes
+        uses: actions/github-script@v4.0.2
```

Comment:
- In the `Check commit messages and descriptions for feat, break, or fix` job, the handling of `COMMIT_DETAILS` could be improved for readability and maintainability.